synthtool/gcp/common.py

```diff
@@ -153,10 +153,22 @@ def _load_partials(self, metadata: Dict):
 
 def decamelize(str: str):
     """ parser to convert fooBar.js to Foo Bar. """
-    str2 = str[0].upper()
-    for chr in str[1:]:
-        if re.match(r"[A-Z]", chr):
-            str2 += " " + chr.upper()
+    prev = " "
+    str2 = ""
```

Comment:
I see this is existing code, but `output` or `decamelized` would be much more readable than `str2`.